dace/codegen/targets/cpu.py

```diff
@@ -1429,7 +1442,7 @@ def generate_nsdfg_arguments(self, sdfg, dfg, state, node):
         inout = set(node.in_connectors.keys() & node.out_connectors.keys())
 
         memlet_references = []
-        for _, _, _, vconn, in_memlet in state.in_edges(node):
+        for _, _, _, vconn, in_memlet in sorted(state.in_edges(node), key=lambda e: e.dst_conn):
```

Comment:
This was a bug fixed by @tbennun so maybe he can comment, but generally we are interested in emitting deterministic code, so if the edges are not otherwise ordered it's a good idea to impose some ordering.